apps/ledger-live-mobile/src/newArch/features/Accounts/Navigator.tsx

```diff
@@ -0,0 +1,152 @@
+    () => ({
+      ...getStackNavigatorConfig(colors, true),
+      headerRight: () => <NavigationHeaderCloseButtonAdvanced onClose={onClose} />,
+    }),
+    [colors, onClose],
+  );
+
+  const onConnectDeviceBack = useCallback((navigation: NavigationProp<Record<string, unknown>>) => {
+    track("button_clicked", {
+      button: "Back arrow",
+      page: ScreenName.ConnectDevice,
+    });
+    navigation.goBack();
+  }, []);
+
+  return (
+    <Stack.Navigator
+      screenOptions={{
+        ...stackNavigationConfig,
+        gestureEnabled: Platform.OS === "ios",
+      }}
+    >
+      {/* Select Crypto (see : apps/ledger-live-mobile/src/screens/AddAccounts/01-SelectCrypto.js) */}
+      <Stack.Screen
+        name={ScreenName.AddAccountsSelectCrypto}
+        component={SelectCrypto}
+        options={{
+          headerLeft: () => <NavigationHeaderBackButton />,
+          headerTitle: "",
+          headerRight: () => <NavigationHeaderCloseButtonAdvanced onClose={onClose} />,
+        }}
+      />
+
+      <Stack.Screen
+        name={ScreenName.SelectNetwork}
+        component={SelectNetwork}
+        options={{
+          headerLeft: () => <NavigationHeaderBackButton />,
+          headerTitle: "",
+          headerRight: () => (
+            <Flex alignItems="center" justifyContent="center" flexDirection="row">
+              {hasClosedNetworkBanner && (
+                <HelpButton eventButton="Choose a network article" url={urls.chooseNetwork} />
+              )}
+              <NavigationHeaderCloseButtonAdvanced onClose={onClose} />
+            </Flex>
+          ),
+        }}
+      />
+
+      {/* Select Accounts */}
+      <Stack.Screen
+        name={ScreenName.SelectAccounts}
+        component={SelectAccounts}
+        options={{
+          headerTitle: "",
+        }}
+      />
+
+      {/* Select Device */}
+      <Stack.Screen
+        name={ScreenName.SelectDevice}
+        component={SelectDevice}
+        options={{
+          headerTitle: () => (
+            <StepHeader
+              subtitle={t("transfer.receive.stepperHeader.range", {
+                currentStep: "2",
+                totalSteps: 3,
+              })}
+              title={t("transfer.receive.stepperHeader.connectDevice")}
+            />
+          ),
+          ...addAccountsSelectDeviceHeaderOptions(onClose),
+        }}
+      />
+
+      {/* Select / Connect Device */}
+      <Stack.Screen
+        name={ScreenName.ConnectDevice}
+        component={ConnectDevice}
+        options={({ navigation }) => ({
+          headerTitle: () => (
+            <StepHeader
+              subtitle={t("transfer.receive.stepperHeader.range", {
+                currentStep: "2",
+                totalSteps: 3,
+              })}
+              title={t("transfer.receive.stepperHeader.connectDevice")}
+            />
+          ),
+          ...connectDeviceHeaderOptions(() => onConnectDeviceBack(navigation)),
+        })}
+      />
```

Comment:
Don't we want to add the following `Stack.Screen` definitions in a dedicated Navigator under `features/DeviceSelection`?

Reply:
didn't knew about that split requirement/recommendation. I'll figure out

Reply:
done ☑️